src/mon/OSDMonitor.cc

```diff
@@ -5733,6 +5733,14 @@ bool OSDMonitor::preprocess_command(MonOpRequestRef op)
     if (auto p = m.find("hostname"); p != m.end()) {
       f->dump_string("host", p->second);
     }
+
+    // try to find logical address
+    if (auto p = m.find("devices"); p != m.end()) {
+      f->dump_string("logical_address", p->second);
+    } else if (auto p = m.find("bluestore_bdev_devices"); p != m.end()) {
```

Comment:
Under what conditions would `devices` key not exist?  IIRC it's not one of the optional keys.

Reply:
You’re right that devices is always present today.  I kept it explicit to preserve the current structure where semantic fields are handled separately from the generic dump loop. This keeps the output contract stable and allows flexibility if the logical address source changes in the future.